packages/expo-cli/src/accounts.ts

```diff
@@ -95,7 +110,165 @@ export async function login(options: CommandOptions): Promise<User> {
   }
 }
 
-async function _usernamePasswordAuth(username?: string, password?: string): Promise<User> {
+/**
+ * Prompt for an OTP with the option to cancel the question by answering empty (pressing return key).
+ */
+async function _promptForOTPAsync(cancelBehavior: 'cancel' | 'menu'): Promise<string | null> {
+  const enterMessage =
+    cancelBehavior === 'cancel' ? 'press Enter to cancel' : 'press Enter for other options';
+  const otpQuestion: Question = {
+    type: 'input',
+    name: 'otp',
+    message: `One-time Password or Backup Code (${enterMessage}):`,
+  };
+
+  const { otp } = await prompt(otpQuestion);
+  if (!otp) {
+    return null;
+  }
+
+  return otp;
+}
+
+/**
+ * Prompt for user to choose a backup OTP method. If selected method is SMS, a request
+ * for a new OTP will be sent to that method. Then, prompt for the OTP, and retry the user login.
+ */
+async function _promptForBackupOTPAsync(
+  username: string,
+  password: string,
+  secondFactorDevices: SecondFactorDevice[]
+): Promise<string | null> {
+  const nonPrimarySecondFactorDevices = secondFactorDevices.filter(device => !device.is_primary);
+
+  if (nonPrimarySecondFactorDevices.length === 0) {
+    throw new Error('No other second-factor devices set up');
```

Comment:
```suggestion
    log.warn(`You need to set up two factor authentication before you can use a OTP. ${log.chalk.dim(learnMore('<link to docs or fyi.expo>'))}`);
    process.exit(1);
```
Maybe create an FYI or link to docs about how to set up a device.

Reply:
This error occurs when the user only has one second factor device set up and chooses the option (`press Enter for more options`) in the prompt. We still want them to be shown this option though even if they only have one second factor device as they could be in a state where they have a second one that is in the process of being setup but is not yet certified/confirmed, in which case this error will help them debug their issue. I'll alter the wording a bit.

Reply:
I'll make a task to add in a link once we have documentation.